app/models/answer.rb

```diff
@@ -0,0 +1,16 @@
+class Answer < ActiveRecord::Base
+  belongs_to :question
+  belongs_to :user
+  has_many :comments, as: :commentable
+  has_many :votes, as: :votable
+
+  def total_votes
+    @vote_total = 0
```

Comment:
You should let the database do this for you
`votes.sum(:vote_amount)` would work